src/core/js/views/buttonsView.js

```diff
@@ -93,8 +93,9 @@ define([
             var buttonState = BUTTON_STATE(changedAttribute);
             if (changedAttribute === BUTTON_STATE.CORRECT || changedAttribute === BUTTON_STATE.INCORRECT) {
                 // Both 'correct' and 'incorrect' states have no model answer, so disable the submit button
-                this.$('.buttons-action').focusNext();
-                this.$('.buttons-action').a11y_cntrl_enabled(false)
+                var $buttonsAction = this.$('.buttons-action');
```

Comment:
Sorry, can you move this variable further up and use it in the `else` block too.

Reply:
sorry